modules/openapi-generator/src/main/resources/nodejsexpresstypescript-server/controllers/Controller.mustache

```diff
@@ -51,6 +54,26 @@ export default class Controller {
   static collectRequestParams(request : any) {
     this.collectFiles(request);
     const requestParams : any= {};
+    // handles any body payload
+    /*
+    if (request.openapi.schema.requestBody !== undefined) {
+    requestParams.body = request.body;
+    }
+    */
+
+    // handles parameters passed through x-www-form-urlencoded
+    if (
+        request.openapi.schema.requestBody !== undefined &&
```

Comment:
only `undefined` check is not enough. The code will throw also an error if you try to access `content` from `null`.
Use `request.openapi.schema.hasOwnProperty('requestBody')` and so on for the other parameters